packages/components/src/select-control/control.js

```diff
@@ -227,38 +246,11 @@ class Control extends Component {
 						icon={ search }
 					/>
 				) }
-				{ inlineTags && <Tags { ...this.props } /> }
-
-				<div className="components-base-control__field">
-					{ !! label && (
-						<label
-							htmlFor={ `woocommerce-select-control-${ instanceId }__control-input` }
-							className="components-base-control__label"
-						>
-							{ label }
-						</label>
-					) }
-					{ this.renderInput() }
-					{ inlineTags && (
-						<span
-							id={ `search-inline-input-${ instanceId }` }
-							className="screen-reader-text"
-						>
-							{ __(
-								'Move backward for selected items',
-								'woocommerce-admin'
-							) }
-						</span>
-					) }
-					{ !! help && (
-						<p
-							id={ `woocommerce-select-control-${ instanceId }__help` }
-							className="components-base-control__help"
-						>
-							{ help }
-						</p>
-					) }
-				</div>
+				{ inlineTags ? (
```

Comment:
This logic (moving field separately) isn't quite correct.  With this logic, `inlineTags` will never get hit inside of `field` since it's not reached.